themis/_index.md

```diff
@@ -1,7 +1,8 @@
 ---
 weight: 1
 bookFlatSection: true
-title: "Themis"
+title: "Themis in a nutshell"
+secondTitle: "Themis"
```

Comment:
Uh... so, which title goes where? Is `secondTitle` our own extension?

I'd guess, the page title (i.e., the browser window, bookmarks, etc.) will say “Themis in a nutshell” while the menu will say “Themis”? Or will it be reverse?

Reply:
@ColliderBang  if it's ours extension, maybe we should rename `secondTitle` into `menuTitle` or similar?

Reply:
Here's how it looks:

![Screen Shot 2020-07-01 at 17 52 14](https://user-images.githubusercontent.com/1256587/86263468-f2875f00-bbc9-11ea-9fa6-21dd5cf7f654.png)

The `title` item still refers to the page title and the page navigation menu title. The new item `productMenuTitle` sets the title in the product menu (at the top).